hooks/environment

```diff
@@ -32,3 +32,4 @@ JSON
 export BUILDKITE_OIDC_TMPDIR=$TMPDIR
 export GOOGLE_APPLICATION_CREDENTIALS=$TMPDIR/credentials.json
 export CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE=$GOOGLE_APPLICATION_CREDENTIALS
+export BUILDKITE_OIDC_AUDIENCE=$AUDIENCE
```

Comment:
can you explain this?

Reply:
Sorry, I was just debugging.

Reply:
https://github.com/elastic/oblt-google-auth-buildkite-plugin/pull/2/commits/c74bd404efef2579000fd0434ea9fe622373bf17 this works now.

it's inspired by https://github.com/google-github-actions/auth/blob/main/src/main.ts#L213